frontend-js.md

```diff
@@ -50,9 +50,14 @@
 
 # Build Tools (Bundlers)
 Module Bundling, on a high level, is a process of integrating together a group of modules in a single file so that multiple modules can be sent to the browser in a single bundle (which is good for performance but bad for debugging).
-- [Webpack](https://webpack.js.org) - a modular build tool 
+- [Webpack](https://webpack.js.org) - a modular build tool
 - [Parcel](https://parceljs.org/) - gives you “blazing fast” bundling as it uses multiple worker processes to ensure that the compilation process is executed in parallel on multiple cores without the need of any configuration
 
+# Module Loader
```

Comment:
May be move it under 'Libraries' section?
And please add the link to the TOC on the top

Reply:
Done